src/__test__/App.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import NavBar from '../components/navbar';
+import '@testing-library/jest-dom';
+
+test('renders navbar without errors', () => {
```

Comment:
**You did a great job! I like the clean code that you wrote!**

- [ ] However, I only can see one test file in your project that is related to the unit test. But, students are meant to write integration tests too. Please implement integration tests in your code. Thank you!

> HINT: You can perform a mocking data fetching and Redux Store connection.

Reply:
Thanks @Rustamxon7 for the feedback. I will work on it 😎